src/react/src/components/Contribute/ProductionLocationInfo.jsx

```diff
@@ -0,0 +1,527 @@
+                                    Sector(s)
+                                </Typography>
+                                <Typography
+                                    component="h4"
+                                    className={classes.subTitleStyles}
+                                >
+                                    Enter the type of products produced at this
+                                    location. For example: Shirts, Laptops,
+                                    Solar Panels.
+                                </Typography>
+                                <StyledSelect
+                                    id="sector"
+                                    name="sector"
+                                    aria-label="Select sector"
+                                    label={null}
+                                    options={
+                                        mapDjangoChoiceTuplesToSelectOptions(
+                                            mockedSectors,
+                                        ) || []
+                                    }
+                                    value={sector}
+                                    onChange={handleSectorChange}
+                                    className={`basic-multi-select notranslate ${classes.selectStyles}`}
+                                    styles={selectStyles}
+                                    placeholder="Select"
+                                    isMulti={!false}
+                                />
+                            </div>
+                            <div className={classes.inputSectionWrapStyles}>
+                                <Typography
+                                    component="h2"
+                                    className={classes.titleStyles}
+                                >
+                                    Product Type(s)
+                                </Typography>
+                                <Typography
+                                    component="h4"
+                                    className={classes.subTitleStyles}
+                                >
+                                    Enter the type of products produced at this
+                                    location. For example: Shirts, Laptops,
+                                    Solar Panels.
+                                </Typography>
+                                <StyledSelect
+                                    creatable
+                                    label={null}
+                                    name="Product Type"
+                                    value={productType}
+                                    onChange={handleProductType}
+                                    placeholder="Enter product type(s)"
+                                    aria-label="Enter product type(s)"
+                                    className={`basic-multi-select notranslate ${classes.selectStyles}`}
```

Comment:
className already exists in the StyledSelect.

Reply:
edited classes